src/os/bluestore/BlueFS.cc

```diff
@@ -65,7 +64,7 @@ uint64_t BlueFS::get_block_device_size(unsigned id)
 
 void BlueFS::add_block_extent(unsigned id, uint64_t offset, uint64_t length)
 {
-  Mutex::Locker l(lock);
```

Comment:
why not use Mutex? I think mutex can provide with some helpful utils like perf counter...and lockdep detection